src/AppBundle/Controller/BlogController.php

```diff
@@ -34,12 +34,18 @@
 class BlogController extends Controller
 {
     /**
-     * @Route("/", name="blog_index")
+     * @Route("/{page}", name="blog_index", defaults={"page" = 1}, requirements={"page" : "\d+"})
```

Comment:
Ok you have a point there I'll working on that, but I think it needs a new @Route